lib/src/react16_suggestors/react_dom_render_migrator.dart

```diff
@@ -107,7 +106,7 @@ class ReactDomRenderMigrator extends GeneralizingAstVisitor
     if (renderFirstArg is InvocationExpression) {
       usage = getComponentUsage(renderFirstArg);
       if (usage != null && usage.componentName == 'ErrorBoundary' ||
```

Comment:
I was looking at this for a sec to make sure you don't need the parenthesis when I realized that I believe it can be simplified. Does
```if (usage?.componentName == 'ErrorBoundary' || renderFirstArg.toSource().startsWith('ErrorBoundary')) {```
work?